iceberg/iceberg-catalog/src/main/java/org/apache/iceberg/hive/HiveSchemaUtil.java

```diff
@@ -134,6 +137,23 @@ public static Type convert(TypeInfo typeInfo) {
     return HiveSchemaConverter.convert(typeInfo, false);
   }
 
+  /**
+   * Produces the difference of two FieldSchema lists by only taking into account the field name and type.
+   * @param subtrahendCollection List of fields to subtract from
```

Comment:
I think minuend and subtrahend are the other way around in this case

Reply:
Woops, yeah..